src/Modules/ConfigOptions.lua

```diff
@@ -1400,10 +1478,83 @@ Sirus adds the following modifiers:
 	{ var = "enemyChaosResist", type = "integer", label = "Enemy ^xD02090Chaos Resistance:", apply = function(val, modList, enemyModList)
 		enemyModList:NewMod("ChaosResist", "BASE", val, "Config")
 	end },
+	{ var = "presetBossSkills", type = "list", label = "Boss Skill Preset", tooltip = [[
+Used to fill in defaults for specific boss skills if the boss config is not set
+
+Bosses' damage is assumed at a 2/3 roll, with no Atlas passives, at the normal monster level for your character level (capped at 84)
+Fill in the exact damage numbers if more precision is needed
+
+Caveats for certain skills are below
+
+Shaper Ball: Allocating Cosmic Wounds increases the penetration to 40% and adds 2 projectiles
+Shaper Slam: Cannot be Evaded.  Allocating Cosmic Wounds doubles the damage and cannot be blocked or dodged
+Maven Memory Game: Is three separate hits, and has a large DoT effect.  Neither is taken into account here.
```

Comment:
Based on feedback this should explicitly state that for EHP hits before death >=4 to survive